saas/settings.py

```diff
@@ -96,6 +96,7 @@
     'BYPASS_PROCESSOR_AUTH': False,
     'CURRENCY_JSON_PATH': os.path.join(os.path.dirname(__file__),
         'static', 'data', 'currencies.json'),
+    'CSV_CUSTOMER_CONTACTS_INCLUDE': True,
```

Comment:
`CSV_CUSTOMER_CONTACTS_INCLUDED` ('d' at the end) or `INCLUDES_CUSTOMER_CONTACT_IN_CSV`. It just sounds strange otherwise.